modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -1,126 +1,141 @@
-= Migrate a live environment to upgrade the version
-:description: Service stopped: Migrate the data, install a new environment, test non-migrated elements behavior and compatibility with new bonita version
+= Updating Bonita Runtime
 
-Service stopped: Migrate the data, install a new environment, test non-migrated elements behavior and compatibility with new bonita version
+:description: This page explains how to update Bonita Runtime (single or multiple nodes) to a newer version of Bonita.
 
-== Overview
+{description}
 
-This page explains how to migrate your platform to a newer version of Bonita.
+== Overview
 
-You can migrate your platform using the `Bonita Migration Tool`. There are 2 versions of the migration tool:
+To update Bonita Runtime you need to use _Bonita Migration Tool_. There are 2 versions of the migration tool:
 
-* The version 1.x that migrate the platform from version 6.0.2 or later to any version up to 7.0.0
-* the version 2.x that migrate the platform from version 7.0.0 or later to any version up to the last one.
+* Version 1.x migrates the elements of Bonita Runtime from version 6.0.2 or later to any version up to 7.0.0
+* Version 2.x migrates the elements of Bonita Runtime from version 7.0.0 or later to any version up to the last one.
 
-For example, if you are migrating from 6.5.3 to 7.1.0, you must migrate from 6.5.3 to 7.0.0 using version 1.x of the _Bonita Migration Tool_, and then migrate from 7.0.0 to 7.1.0 using version 2.x.
+For example, if you consider updating from Bonita 6.5.3 to Bonita 7.1.0, you must first update from Bonita 6.5.3 to Bonita 7.0.0 using version 1.x of Bonita Migration Tool, and then update from Bonita 7.0.0 to Bonita 7.1.0 using version 2.x. +
+You are recommended not to start Bonita 7.0.0 after you update to it, but to proceed immediately with the second phase of the update.
 
 [CAUTION]
 ====
-You should always migrate to the latest version of a major version. Usually, maintenance version might contain bug fixes made after the release of subsequent major or minor release.
+You should always update to the latest version of a main version, even if not the latest Bonita version. +
```

Comment:
the latest version of a main version, : I find it confusing... can we call it "the latest version of a main release"?

Reply:
ok